src/index.scss

```diff
@@ -1,6 +1,9 @@
 @import "~@newamerica/data-table/dist/styles.css";
 @import "~@newamerica/components/dist/styles.css";
 @import "~@newamerica/meta/dist/styles.css";
+
+$dashboard__width: 1200px;
```

Comment:
tbh this is not the same as the width of the site when you consider padding, but I didn't change it yet